src/layouts/base-new/base-new-layout.module.css

```diff
@@ -65,7 +65,7 @@ Asana task: https://app.asana.com/0/1202114367927919/1202316169578550/f
   }
 
   /* On Desktop, footer should layer above sidebar */
-  @media (--dev-dot-tablet-up) {
+  @media (--dev-dot-desktop) {
```

Comment:
Fixing an earlier error of mine - as indicated by the comment, this should be `desktop`, not `tablet-up`.